tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35,py36,py37,py38
+envlist = py,extra-deps
```

Comment:
Heh, I'm still now sure what's the best approach. If you do have all environments installed, and want to test the code against them all (which is a good thing to do), you now need to pass env list to tox - it is unclear which one, and the list might be changing over time. So for this use case not having a list of envs to test against looks less developer friendly.